vault/data_source_ldap_static_role_credentials_test.go

```diff
@@ -54,15 +54,15 @@ resource "vault_ldap_secret_backend" "test" {
 }
 
 resource "vault_ldap_secret_backend_static_role" "role" {
-  path = vault_ldap_secret_backend.test.path
+  mount = vault_ldap_secret_backend.test.path
```

Comment:
I think we will need to update these in the dynamic role tests as well

Reply:
yep, i can do that in the next PR, which adds a data source for dynamic creds.